tests/integration_tests/test_config_files.py

```diff
@@ -1,136 +1,136 @@
-
-    # test store_a_copy_of_zeek_files
-    assert "zeek_files" in os.listdir(output_dir)
-
-    # test metadata_dir
-    assert "metadata" in os.listdir(output_dir)
-    metadata_path = os.path.join(output_dir, "metadata")
-    for file in ("test.conf", "whitelist.conf", "info.txt"):
-        assert file in os.listdir(metadata_path)
-
-    # test label=malicious
-    assert int(database.get_label_count("malicious")) > 370
-
-    # test disable
-    for module in ["template", "ensembling", "Flow ML Detection"]:
-        assert module in database.get_disabled_modules()
-
-    shutil.rmtree(output_dir)
-
-
-@pytest.mark.parametrize(
-    "pcap_path, expected_profiles, output_dir, redis_port",
-    [
-        (
-            "dataset/test8-malicious.pcap",
-            1,
-            "pcap_test_conf2/",
-            6668,
-        )
-    ],
-)
-def test_conf_file2(pcap_path, expected_profiles, output_dir, redis_port):
-    """
-    In this test we're using tests/test2.conf
-    """
-
-    output_dir = create_output_dir(output_dir)
-    output_file = os.path.join(output_dir, "slips_output.txt")
-    command = (
-        f"./slips.py "
-        f"-t  -e 1 "
-        f"-f {pcap_path} "
-        f"-o {output_dir} "
-        f"-c tests/integration_tests/test2.conf "
-        f"-P {redis_port} "
-        f"> {output_file} 2>&1"
-    )
-    # this function returns when slips is done
-    os.system(command)
-
-    assert has_errors(output_dir) is False
-
-    shutil.rmtree(output_dir)
+"""
+This file tests 2 different config files other than slips' default config/slips.conf or onfig/slips.yaml
+test/test.yaml and tests/test2.yaml
+"""
+
+from slips.main import Main
+from tests.common_test_utils import (
+    is_evidence_present,
+    create_output_dir,
+    has_errors,
+    check_for_text,
+)
+from tests.module_factory import ModuleFactory
+import pytest
+from ...slips import *
+import shutil
+import os
+
+
+alerts_file = "alerts.log"
+
+
+def create_Main_instance(input_information):
+    """returns an instance of Main() class in slips.py"""
+    main = Main(testing=True)
+    main.input_information = input_information
+    main.input_type = "pcap"
+    main.line_type = False
+    return main
+
+
+@pytest.mark.parametrize(
+    "pcap_path, expected_profiles, output_dir, redis_port",
+    [
+        (
+            "dataset/test7-malicious.pcap",
+            290,
+            "test_configuration_file/",
+            6667,
+        )
+    ],
+)
+def test_conf_file(pcap_path, expected_profiles, output_dir, redis_port):
+    """
+    In this test we're using tests/test.conf
+    """
+    output_dir = create_output_dir(output_dir)
+    output_dir = create_output_dir(output_dir)
+
+    output_file = os.path.join(output_dir, "slips_output.txt")
+    command = (
+        f"./slips.py "
+        f"-t  -e 1 "
+        f"-f {pcap_path} "
+        f"-o {output_dir} "
+        f"-c /StratosphereLinuxIPS/tests/integration_tests/test.yaml "

```

Comment:
why did you add /StratosphereLinuxIPS/ here? it should be relative, should start with tests/